packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 4.5.0
 
+* Adds support to show JavaScript dialog. See `WebViewController.setOnJavaScriptAlertDialog`, `WebViewController.setOnJavaScriptConfirmDialog` and `WebViewController.setOnJavaScriptTextInputDialog`.
```

Comment:
Nit: "Adds support for custom handling of JavaScript dialogs. See [...]"

Reply:
(Also, please wrap to ~80 characters.)

Reply:
This two lines can be removed you have the content now in 4.6.0